code/debris/debris.cpp

```diff
@@ -480,6 +481,15 @@ object *debris_create(object *source_obj, int model_num, int submodel_num, vec3d
 
 	float radius = submodel_get_radius(db->model_num, db->submodel_num);
 
+	// if its generic, maybe cull it if its too small and far
+	if (!hull_flag) {
+		// Make vaporize debris seen from farther away
+		float dist = vm_vec_dist_quick(pos, &Eye_position);
+		if (dist > radius * 200.0f) {
+			return NULL;
```

Comment:
nullptr at least for new lines.  If you want to update the rest of this function with nullptr to match, I don't think anyone else will complain.

Everything else looks good.